ITC205_BaseLine_Code/src/BorrowBookControl.java

```diff
@@ -26,12 +26,12 @@ public void setUI(BorrowBookUI ui) {
 			throw new RuntimeException("BorrowBookControl: cannot call setUI except in INITIALISED state");
 			
 		this.ui = ui;
-		ui.setState(BorrowBookUI.UI_STATE.READY);
+		ui.setState(BorrowBookUI.UIState.READY);
 		state = CONTROL_STATE.READY;		

```

Comment:
D1 change CONTROL_STATE to ControlState throughout file (Find/replace)

Reply:
As CONTROL_STATE is only in BorrowBookControl.java it will be fixed in that branch